resources/resources.qrc

```diff
@@ -10,6 +10,7 @@
     <file>play_icon.png</file>
     <file>no_publishes_found.png</file>
     <file>no_item_selected_512x400.png</file>
+    <file>too_many_publishes_512x400.png</file>
     <file>folder_512x400.png</file>
     <file>loading_512x400.png</file>
     <file>loading_100x100.png</file>
```

Comment:
Can't comment on image below, so commenting here. The light gray text reads:

Please select a Publish to see more details.